code/obj/item/plants.dm

```diff
@@ -484,6 +482,25 @@
 		..()
 		return
 
+/obj/item/plant/flower/holorose
```

Comment:
This should probably inherit from rose, not be its own separate subtype. That also lets you have a `get_possible_names` proc and not duplicate all that code in `New`.

Reply:
x

Reply:
Think I've sorted it out, let me know if there's something I did wrong or oculd be improved!!